registry/tracecat_registry/templates/detect/lookup_ip_address/ipinfoio.yml

```
@@ -0,0 +1,26 @@
+type: action
+definition:
+  name: lookup_ip_address
+  namespace: tools.ipinfoio
+  title: Lookup IP address
+  description: Get IPinfo.io report for an IP address
+  display_group: IPinfo.io
+  doc_url: https://ipinfo.io/developers
+  secrets:
+    - name: ipinfoio
+      keys:
+        - API_KEY

```

```suggestion
        - IPINFO_API_TOKEN

```
Better aligns with IPInfo docs (which calls it token vs key).